src/libraries/System.Private.CoreLib/src/System/TimeZoneInfo.Unix.NonAndroid.cs

```diff
@@ -613,7 +613,7 @@ private static string GetTimeZoneDirectory()
         {
             string? tzDirectory = Environment.GetEnvironmentVariable(TimeZoneDirectoryEnvironmentVariable);
 
-            if (tzDirectory == null)
+            if (tzDirectory == null || srting.IsNullOrEmpty(tzDirectory))
```

Comment:
Nit: the `== null` is covered in `IsNullOrEmpty` and redundant.

Reply:
Oh!! Thank you :)